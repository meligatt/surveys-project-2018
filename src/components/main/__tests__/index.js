import React from 'react';
import renderer from 'react-test-renderer';
import Main from '../index';

describe('Main', () => {
   it('renders correctly', () => {
     const tree = renderer.create(
       <Main>
         <div>some</div>
         <div>children</div>
       </Main>
    ).toJSON();

    expect(tree).toMatchSnapshot();
   })
});
