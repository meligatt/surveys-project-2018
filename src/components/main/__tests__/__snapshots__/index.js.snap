// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main renders correctly 1`] = `
<main
  className="main"
  role="main"
>
  <div
    className="main__container"
  >
    <div
      className="main__content"
    >
      <div>
        some
      </div>
      <div>
        children
      </div>
    </div>
  </div>
</main>
`;
